package/requirements.txt

```diff
@@ -9,7 +9,7 @@ watchgod~=0.8.2
 plotly>=4.0
 pandas>=0.24
 sqlalchemy>=1.4, <3
-strawberry-graphql>=0.99.0, <1.0
+strawberry-graphql[cli]>=0.99.0, <0.185  # Pinning as recent versions don't work with mypy < 1.0 
```

Comment:
Also, just wondering, was the issue mypy? I pinned it <0.184.1 (very similar) in #1407 to avoid https://github.com/strawberry-graphql/strawberry/commit/8d97bd61383af330c05e6b578cec55ac768b6e9e, which brings in Typer.

In any case, should relax it at some point to start picking up newer 🍓 again.

Reply:
Yup, this issue is mypy failing with pydantic -- https://github.com/kedro-org/kedro-viz/pull/1311 and haven't figured best fix that doesn't involve a big refactor.